### PR TITLE
Corpse description snippets

### PR DESCRIPTION
**Pull request by faefux**
#### Summary
Content "snippets for corpses to allow for some  cosmetic variation."

#### Purpose of change
Seen the different kinds of corpses when using them and decided corpses should have snippets to allow for variation.

#### Describe the solution
Wrote some quick and short snippets in the hopes that other will come to add some too.

#### Describe alternatives you've considered

Just not I guess.
#### Testing
I scrolled through them in the item debug menu to see the different tags. Everything looks in order.

#### Additional context
The purpose of this was mostly to encourage others to write some interesting ones too both for the corpses i gave the `"snippet_category"` and otherwise.  So feel free to have at it.

### CODE REVIEW

**Comment by actual-nh** [comment]
Ping: @Venera3?

File: data/json/items/corpses/corpses.json [hunk 1]
```diff
@@ -101,7 +101,17 @@
...
-    "material": [ "hflesh" ]
+    "material": [ "hflesh" ],
+    "snippet_category": [
+      {
+        "id": "child_gunned_01",
+        "text": "The dead body of a child, riddled by bullets to the extent that you can no longer tell their gender."
+      },
+      {
+        "id": "child_gunned_02",
+        "text": "The corpse of an infant.  A single small calibre bullet has been put through its skull."
```
Comment (Night-Pryanik) [inline data/json/items/corpses/corpses.json:112]:
```suggestion
        "text": "The corpse of an infant.  A single small-caliber bullet has been put through its skull."
```

File: data/json/items/corpses/corpses.json [hunk 2]
```diff
@@ -149,7 +159,26 @@
...
+        "id": "generic_stabbed_02",
+        "text": "The bloody corpse of a young man.  He has been stabbed repeatedly in the jugular and throat with a small blade."
+      },
+      {
+        "id": "generic_stabbed_03",
+        "text": "The dead body of a person.  They look to have been skewered through with a large sharp implement."
+      },
+      {
+        "id": "generic_stabbed_04",
+        "text": "The corpse of a tall middle aged woman.  She appears to have many stab wounds up her legs and thighs."
```
Comment (Night-Pryanik) [inline data/json/items/corpses/corpses.json:175]:
```suggestion
        "text": "The corpse of a tall middle-aged woman.  She appears to have many stab wounds up her legs and thighs."
```

File: data/json/items/corpses/corpses.json [hunk 3]
```diff
@@ -183,5 +212,47 @@
...
+    "weight": "52 kg",
+    "material": [ "hflesh" ],
+    "snippet_category": [
+      {
+        "id": "generic_gunned_01",
+        "text": "The dead body of a person.  Their forehead bears a large bullet entrance wound.  An even larger exit wound is present on the back of their head."
+      },
+      {
+        "id": "generic_gunned_02",
+        "text": "The corpse of a young blond woman shot several times in the back with a low calibre weapon."
```
Comment (Night-Pryanik) [inline data/json/items/corpses/corpses.json:233]:
```suggestion
        "text": "The corpse of a young blond woman shot several times in the back with a low-caliber weapon."
```

File: data/json/items/corpses/corpses.json [hunk 4]
```diff
@@ -183,5 +212,47 @@
...
+      },
+      {
+        "id": "generic_gunned_03",
+        "text": "An androgynous looking corpse gored by arms fire chest to thighs, leaving the face eerily pristine."
```
Comment (Night-Pryanik) [inline data/json/items/corpses/corpses.json:237]:
```suggestion
        "text": "An androgynous-looking corpse gored by arms fire chest to thighs, leaving the face eerily pristine."
```

File: data/json/items/corpses/corpses.json [hunk 5]
```diff
@@ -183,5 +212,47 @@
...
+      },
+      {
+        "id": "generic_gunned_04",
+        "//": "canoeing",
+        "text": "The corpses of a seemingly older male.  He appears to have been shot in the forehead at a very close range leaving you a good view of his brain matter."
```
Comment (Night-Pryanik) [inline data/json/items/corpses/corpses.json:242]:
```suggestion
        "text": "The corpse of a seemingly older male.  He appears to have been shot in the forehead at a very close range leaving you a good view of his brain matter."
```

File: data/json/items/corpses/corpses.json [hunk 6]
```diff
@@ -183,5 +212,47 @@
...
+      },
+      {
+        "id": "generic_gunned_05",
+        "text": "The corpse of a middle aged woman, her legs and arms covered in blood.  She appears to have been shot in the thigh."
```
Comment (Night-Pryanik) [inline data/json/items/corpses/corpses.json:246]:
```suggestion
        "text": "The corpse of a middle-aged woman, her legs and arms covered in blood.  She appears to have been shot in the thigh."
```

**Comment by faefux** [comment]
Thank you, @Night-Pryanik!

**Pull request merged**